app/src/main/res/drawable/splash_background.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item android:drawable="@color/white"/>
+
+    <!--<item
+        android:height="200dp"
+        android:width="400dp"
+        android:drawable="@drawable/mifos_logo"
+        android:gravity="center"/>-->
+
+    <item>
+        <bitmap
```

Comment:
Using default dimensions, logo is becoming too big for Splash Screen.
![screenshot_2018-06-03-11-34-39-372_org mifos mobilebanking](https://user-images.githubusercontent.com/22195621/40887855-7c4fa846-676c-11e8-93d2-f55ccf666a60.png)